LICENSE

```diff
@@ -19,3 +19,29 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+******************************************************************************
+
+This project was derived from the project
+https://github.com/miguelgrinberg/oreilly-flask-apis-video
+
+The MIT License (MIT)
+
+Copyright (c) 2014 Miguel Grinberg
```

Comment:
The whole app architecture is derived from https://github.com/miguelgrinberg/oreilly-flask-apis-video but it wasn't a fork. I essentially rebuilt the app with his structure commit-by-commit (and mention that in the commit messages).